### PR TITLE
20.9.1 rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.9.1.md [hunk 1]
```diff
@@ -0,0 +1,1203 @@
...
+
+##### JSONtoCSV  
+Maintenance and stability enhancements.
+
+##### DeleteContext  
+Updated the script to execute using the DBot role.
+
+---
+
+### Common Types Pack v1.9.0
```
Comment (jochman) [inline content-repo/extra-docs/releases/20.9.1.md:231]: remove this

File: content-repo/extra-docs/releases/20.9.1.md [hunk 2]
```diff
@@ -0,0 +1,1203 @@
...
+
+### Elasticsearch Feed Pack v1.0.6
+#### Integrations
+##### Elasticsearch Feed  
+- Changed the default ***Feed Type*** from **Cortex XSOAR Feed** to **Cortex XSOAR MT Shared Feed**.
+- Updated the Docker image.
+
+---
+
+### Email Communication Pack v1.2.2
```
Comment (jochman) [inline content-repo/extra-docs/releases/20.9.1.md:773]: what is the change in the layouts?

File: content-repo/extra-docs/releases/20.9.1.md [hunk 3]
```diff
@@ -0,0 +1,1203 @@
...
+
+##### RiskIQPassiveTotalTrackersScript  
+Enhancement script to enrich web trackers information for Domain and IP type of indicators.
+
+##### RiskIQPassiveTotalWhoisScript  
+Enhancement script to enrich whois information for Domain and Email type of indicators.
+
+---
+
+### Pentera Pack v1.0.2 (Partner Supported)
```
Comment (jochman) [inline content-repo/extra-docs/releases/20.9.1.md:1023]: what is the change in the layouts?

**Review by jochman — CHANGES_REQUESTED** [review]
There are lots of empty 'layouts' changelogs. Check if it's a problem that we're not populating those changes.

**Review by glicht — CHANGES_REQUESTED** [review]
We have some problem in the right side toc: 

image

File: content-repo/extra-docs/releases/20.9.1.md [hunk 4]
```diff
@@ -0,0 +1,1205 @@
...
+
+---
+
+### Common Types Pack v1.9.0
+
+
+---
+
+### Common Widgets Pack v1.0.1
+#### GetLargestInputsAndOuputsInIncidents
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.1.md:237]: What changed here? Is this new or updated?

Comment (moishce) [reply content-repo/extra-docs/releases/20.9.1.md:237]: It's new, but only from version 6.0.0

Comment (jochman) [reply content-repo/extra-docs/releases/20.9.1.md]: so you can remove it

Comment (moishce) [reply content-repo/extra-docs/releases/20.9.1.md]: I deleted it already

File: content-repo/extra-docs/releases/20.9.1.md [hunk 5]
```diff
@@ -0,0 +1,1187 @@
...
+##### Azure Log Analytics (Beta)  
+Log Analytics is a service that helps you collect and analyze data generated by resources in your cloud and on-premises environments.
+#### Playbooks
+##### Azure Log Analytics - Query From Saved Search  
+Executes a query from a saved search in Azure Log Analytics.
+
+
+---
+
+### New: MicrosoftCloudAppSecurity Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:14]:
```suggestion
### New:  Microsoft Cloud App Security   Pack v1.0.0
```

File: content-repo/extra-docs/releases/20.9.1.md [hunk 6]
```diff
@@ -0,0 +1,1187 @@
...
+---
+
+### Demisto REST API Pack v1.1.1
+#### Scripts
+##### DemistoUploadFile  
+Maintenance and stability enhancements.
+
+---
+
+### Deprecated Content Pack v1.6.5
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:277]: lets remove deprecated pack

File: content-repo/extra-docs/releases/20.9.1.md [hunk 7]
```diff
@@ -0,0 +1,1187 @@
...
+
+### MongoDB Pack v1.1.0
+#### Integrations
+##### MongoDB  
+- Fixed an issue where it was not possible to apply filters in the *_id* field.
+- Added the *sort* argument to the ***mongodb-query*** command.
+
+---
+
+### Non Supported Pack v2.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:886]: lets remove non supported

File: content-repo/extra-docs/releases/20.9.1.md [hunk 8]
```diff
@@ -0,0 +1,1187 @@
...
+  - *File Entry Tag*
+  - *Work Note Entry Tag*
+  - *Comment Entry Tag*
+
+#### Integrations
+##### ServiceNow v2  
+- Maintenance and stability enhancements.
+- Fixed a minor bug in the ***servicenow-create-ticket*** command.
+
+##### ServiceNow (Deprecated)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:1068]: lets remove

File: content-repo/extra-docs/releases/20.9.1.md [hunk 9]
```diff
@@ -0,0 +1,1187 @@
...
+
+---
+
+### ThreatConnect Pack v2.0.8
+#### Integrations
+##### ThreatConnect v2  
+- Added support for fetching indicators from multiple owners.
+- The DBotScore calculation now works as expected.
+
+##### ThreatConnect (Deprecated)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:1112]: lets remove

File: content-repo/extra-docs/releases/20.9.1.md [hunk 10]
```diff
@@ -0,0 +1,1187 @@
...
+### illuminate Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### illuminate  
+Updated the Content Pack support information.
+
+---
+
+### okta Pack v1.0.4
+#### Integrations
+##### okta (Deprecated)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:1175]: lets remove

File: content-repo/extra-docs/releases/20.9.1.md [hunk 11]
```diff
@@ -0,0 +1,1187 @@
...
+---
+
+### okta Pack v1.0.4
+#### Integrations
+##### okta (Deprecated)  
+Maintenance and stability enhancements.
+
+---
+
+### <span>urlscan.i</span>o Pack v1.0.2
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.1.md:1180]: whats with the `span` here?

**Review by glicht — APPROVED** [review]

**Review by jochman — APPROVED** [review]

**Pull request merged**